File: backend/app/Models/Hero/Ability.php
<?php

namespace App\Models\Hero;

use Eloquent;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

/**
 * @mixin Eloquent
 * @mixin Builder
 */
class Ability extends Model
{
  protected $fillable = [
    'name',
    'description',
    'photo_url',
    'hero_id',
  ];

  public function hero(): BelongsTo
  {
    return $this->belongsTo(Hero::class);
  }
}

File: UserPanel/Models/Ability.cs
using System.Text.Json.Serialization;
namespace UserPanel.Models;
public class Ability
{
    [JsonPropertyName("id")] public required int Id { get; set; }
    [JsonPropertyName("name")] public required string Name { get; set; }
    [JsonPropertyName("description")] public required string Description { get; set; }
    [JsonPropertyName("photo_url")] public required string PhotoUrl { get; set; }
}